config/settings_data.json

```diff
@@ -42,17 +42,28 @@
       "inputs_shadow_horizontal_offset": 0,
       "inputs_shadow_vertical_offset": 4,
       "inputs_shadow_blur": 5,
-      "card_style": "standard",
-      "card_image_padding": 0,
-      "card_text_alignment": "left",
-      "card_color_scheme": "background-2",
-      "card_border_thickness": 0,
-      "card_border_opacity": 10,
-      "card_corner_radius": 0,
-      "card_shadow_opacity": 0,
-      "card_shadow_horizontal_offset": 0,
-      "card_shadow_vertical_offset": 4,
-      "card_shadow_blur": 5,
+      "product_card_style": "standard",
+      "product_card_image_padding": 0,
+      "product_card_text_alignment": "left",
+      "product_card_color_scheme": "background-2",
+      "product_card_border_thickness": 0,
+      "product_card_border_opacity": 10,
+      "product_card_corner_radius": 0,
+      "product_card_shadow_opacity": 0,
+      "product_card_shadow_horizontal_offset": 0,
+      "product_card_shadow_vertical_offset": 4,
+      "product_card_shadow_blur": 5,
+      "blog_card_style": "standard",
+      "blog_card_image_padding": 0,
+      "blog_card_text_alignment": "left",
+      "blog_card_color_scheme": "background-2",
+      "blog_card_border_thickness": 0,
+      "blog_card_border_opacity": 10,
+      "blog_card_corner_radius": 0,
+      "blog_card_shadow_opacity": 0,
+      "blog_card_shadow_horizontal_offset": 0,
+      "blog_card_shadow_vertical_offset": 4,
+      "blog_card_shadow_blur": 5,
```

Comment:
I know that we don't have much control over how themes are updated etc. but I wonder if we'll encounter any issues when we update our themes with the above new settings 🤔

Reply:
Thanks for actioning this, Sofia! I'm going to "unresolve" this conversation simply for visibility 👍🏼